docs/modules/ROOT/pages/create.adoc

```diff
@@ -0,0 +1,70 @@
+== How to Create a Starter Kit
+In this guide, you'll learn how to create your own starter kit. Not sure what starter kits are? Check out https://docs.openzeppelin.com/starter-kits/2.3/[getting started].
+
+=== Why Should I Create a Starter Kit?
+It is still incredibly hard for new developers to enter the Ethereum ecosystem compare to Web2. New developers suffer from a lack of documentation, immature tools, and a huge upfront commitment. Starter kits represent a bridge between Web2 and Web3 supercharged to bootstrap developers into the Ethereum ecosystem within minutes using familiar concepts and technologies, such as React, NPM, Mocha while introducing Web3 components Infura, OpenZeppelin, Truffle, Rimble.
```

Comment:
```suggestion
It is still very hard for new developers to enter the Ethereum ecosystem compared to Web2. New developers suffer from a lack of comprehensive documentation and a huge upfront commitment. Starter kits represent a bridge between Web2 and Web3 supercharged to bootstrap developers into the Ethereum ecosystem within minutes, using familiar concepts and technologies, such as React, NPM, and Mocha, while introducing Web3 components like Infura, OpenZeppelin, Truffle, and Rimble.
```

Reply:
It feels odd to mention immature tools and poor documentation in the documentation site of a tool